projects/output-and-error-handling/parseRetryValue_test.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestParseRetryValue(t *testing.T) {
```

Comment:
This would probably make a good table-driven test like your parseResponse test is :) All the tests seem to do the same thing just with different values, so being able to just look at a table of expected inputs and outputs is quicker and easier to understand and maintain than having several copies of ~the same tests